packages/did-core-test-server/suites/did-spec/defaultSuiteConfig.json

```diff
@@ -1,6 +1,49 @@
 {
   "name": "did-spec",
   "didMethods": {
+    "did:key - 2018 Ed25519 cryptosuite - did-key-js - Digital Bazaar": {
```

Comment:
@OR13 -- I expect we're going to have to label things like this given that there will be multiple implementations and types of did:key tests. Thoughts?

Reply:
yeah, probably there is better ways of handling this, but for now, this is good.